### PR TITLE
Add an unlisted metadata property

### PR DESCRIPTION
**Pull request by amir**
Addresses #158

### CODE REVIEW

**Comment by srid** [comment]
@amir Does this still generate the .html for the unlisted zettel?

**Comment by amir** [comment]
@srid yes, it does.

File: neuron/src/app/Neuron/Reader/Org.hs [hunk 1]
```diff
@@ -39,8 +40,12 @@ extractMetadata doc
...
         tags = fmap Tag . words <$> lookup "tags" properties
+        unlisted = parseUnlisted <$> lookup "unlisted" properties
     pure $ Just Meta {..}
   | otherwise = pure Nothing
   where
     parseDate :: Text -> Either ZettelParseError Day
     parseDate date = maybeToRight (Tagged $ "Invalid date format: " <> date) $ parseZettelDate @Maybe date
+
+    parseUnlisted :: Text -> Bool
+    parseUnlisted a = toLower a == "true"
```
Comment (srid) [inline neuron/src/app/Neuron/Reader/Org.hs:51]: @felko Does this look right? I'm not familiar with org.

**Comment by srid** [comment]
Merged in #325

**Pull request closed without merging**